configuration.md

```diff
@@ -13,6 +13,7 @@ Casual supports the following configuration options which can be set using envir
     See [Inbound Startup Configuration](inbound.md#startup-configuration) for more details.
 * `CASUAL_OUTBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO.
 * `CASUAL_INBOUND_USE_EPOLL` - If set to true, Netty uses epoll instead of NIO.
+* `CASUAL_INBOUND_INITIAL_DELAY` - Delay the startup of the inbound server. Default is no delay.
```

Comment:
In what unit of measure? Seconds, milliseconds?

Reply:
Bananas?
I'll add seconds :)